packages/server/src/server/services/findMyService/index.ts

```diff
@@ -50,16 +52,40 @@ export class FindMyService {
         return await FindMyService.getFriends();
     }
 
-    static getDevices(): NodeJS.Dict<any> | null {
-        const devicesPath = path.join(FileSystem.findMyDir, 'Devices.data');
-        if (!fs.existsSync(devicesPath)) return null;
-
-        const data = fs.readFileSync(devicesPath, { encoding: 'utf-8' });
+    private static readDataFile<T extends "Devices" | "Items">(
+        type: T
+    ): Promise<Array<T extends "Devices" ? FindMyDevice : FindMyItem> | null> {
+        const devicesPath = path.join(FileSystem.findMyDir, `${type}.data`);
+        return new Promise((resolve, reject) => {
+            fs.readFile(devicesPath, { encoding: "utf-8" }, (err, data) => {
```

Comment:
I switched to the async version of `readFile` so we can hit the disk in parallel while we access both data files.